.github/workflows/scripts/kebab-case.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+non_compliant_files=()
+ignoreList=("^\.\/.git" "^\.\/\..*" "^\.\/[^\/]*$")
+ignoreList+=("^\.\/node_modules")
+while IFS= read -r line; do
+ignoreList+=(".*$line")
+done < .gitignore
+while read -r file; do
+basefile=$(basename "$file")
+ignoreFile=false
+for pattern in "${ignoreList[@]}"; do
+    if [[ "$file" =~ $pattern ]]; then
+    ignoreFile=true
+    break
+    fi
+done
+if $ignoreFile; then
+    continue
+elif ! echo "$basefile" | grep -q -E "^([a-z0-9]+-)*[a-z0-9]+(\.[a-zA-Z0-9]+)?$|^([a-z0-9]+_)*[a-z0-9]+(\.[a-zA-Z0-9]+)?$"; then
+    non_compliant_files+=("$file")
+    echo "::warning file=$file::This file is not in kebab-case or snake_case"
+fi
+done < <(find . -type f -name '*.ts' -print | grep -E '/[a-z]+[a-zA-Z]*\.ts$')
+if [ ${#non_compliant_files[@]} -ne 0 ]; then
+echo "The following files are not in kebab-case or snake_case:"
```

Comment:
I thought I fixed this to not include snake_case. We should only support kebab-case. Also you need to change the execution permissions on this file as GitHub Actions failed to run.

Reply:
I took it from the pay.ubq repo that seemed the most up the date and Ty I was wondering wtf was going on with it will update shortly